packages/plugins/AddressSearch/README.md

```diff
@@ -120,6 +120,7 @@ These fields are interpreted by all implemented services.
 | patterns | string[]? | Allows specifying input patterns. In a single-field search, a pattern can be as easy as `{{theWholeThing}}`, where `theWholeThing` is also the feature field name to search in. In more complex scenarios, you may add separators and multiple fields, e.g. `{{gemarkung}} {{flur}} {{flstnrzae}}/{{flstnrnen}}` would fit many parcel search services. Mutually exclusive to `fieldName`. |
 | patternKeys | Record<string, string>? | Maps field names from patterns to regexes. Each field name has to have a definition. Each regex must have one capture group that is used to search. Contents next to it are ignored for the search and just used for matching. E.g. `'([0-9]+)$'` would be a value for a key that fits an arbitrary number string at the input's end. |
 | srsName | string? | Name of the projection (srs) for the query. |
+| likeFilterAttributes | Record<string, string>? | Key/value pairs that are mapped to custom attributes of the isLike operators used in filters. The default is {wildCard: "\*",singleChar: ".",escapeChar: "!"} which results in <ogc:PropertyIsLike wildCard="*" singleChar="." escapeChar="!"> |
```

Comment:
As mentioned in the comment in `packages/lib/getFeatures/wfs/buildWfsFilter.ts`, the key words may vary.
I think this should be reflected here as well.

Also, it should be written that all three parameters have to be defined on the object.

Also also, I've added a [new ADR](https://github.com/Dataport/polar/wiki/Architecture-Decision-Records-%F0%9F%93%83#2024-08-05-configuration-parameters-in-tables-have-to-be-ordered-by-a-required-and-b-alphabetically) regarding the order of parameters in the documentation.

Reply:
> As mentioned in the comment in packages/lib/getFeatures/wfs/buildWfsFilter.ts, the key words may vary.
I think this should be reflected here as well.

It is mentioned here: "**Key**/**value** pairs are mapped to custom attributes". So the key and the value is mapped.

> Also, it should be written that all three parameters have to be defined on the object.

You don't have to define all three parameters.

> Also also, I've added a [new ADR](https://github.com/Dataport/polar/wiki/Architecture-Decision-Records-%F0%9F%93%83#2024-08-05-configuration-parameters-in-tables-have-to-be-ordered-by-a-required-and-b-alphabetically) regarding the order of parameters in the documentation.

I will adjust it.

Reply:
📞 ✔️